ocamlorg.opam

```diff
@@ -100,10 +92,10 @@ build: [
   ]
 ]
 pin-depends: [
+  ["river.dev" "git+https://github.com/kayceesrk/river#master"]
+  ["hilite.dev" "git+https://github.com/tmattio/hilite#main"]
```

Comment:
Is it normal that it point to your fork, while the one [here ](https://github.com/ocaml/ocaml.org/pull/420/files#diff-a6bb1b785ef3eee7ba6830286c89e9ca4b09b71218f00dc3830faef96c3aa6e2R18)point to the original repo?

When I `make build` locally, dune produces the following change for me:

```diff
diff --git a/ocamlorg.opam b/ocamlorg.opam
index 00d797e..226f849 100644
--- a/ocamlorg.opam
+++ b/ocamlorg.opam
@@ -93,7 +93,7 @@ build: [
 ]
 pin-depends: [
   ["river.dev" "git+https://github.com/kayceesrk/river#master"]
-  ["hilite.dev" "git+https://github.com/tmattio/hilite#main"]
+  ["hilite.dev" "git+https://github.com/patricoferris/hilite#main"]
   ["esbuild.dev" "git+https://github.com/tmattio/opam-esbuild#main"]
   ["tailwindcss.dev" "git+https://github.com/tmattio/opam-tailwindcss#main"]
   ["code-mirror.dev" "git+https://github.com/patricoferris/jsoo-code-mirror#static"]
```